app/controllers/catalog_controller.rb

```diff
@@ -222,7 +222,7 @@ def resolve_doi
     raise Blacklight::Exceptions::RecordNotFound unless params.key?(:doi)
 
     doi_query = params[:doi]
-    query = { q: "referenced_by_ssim:*\"#{doi_query}\"" }
+    query = { q: "uri_tesim:*\"#{doi_query}\"" }
```

Comment:
Can/should we make this field a string rather than text (e.g. `uri_s`)? I am afraid the tokenization that happens with text fields (`*_tesim`) might give us false positives in the long term.

Reply:
or `uri_ssim`